<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="bottomMargin">
         <number>290</number>
        </property>
        <item>
         <widget class="QPushButton" name="viewRecipes">
          <property name="text">
           <string>View Recipes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addRecipe">
          <property name="text">
           <string>Add Recipe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="viewIngredients_2">
          <property name="text">
           <string>View Ingredients</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addIngredients">
          <property name="text">
           <string>Add Ingredient</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="viewRecipe">
         <widget class="QListWidget" name="listOfRecipes">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>120</y>
            <width>341</width>
            <height>261</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>70</y>
            <width>113</width>
            <height>24</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>70</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Search:</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="viewIngredients">
         <widget class="QListWidget" name="listOfIngredients">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>120</y>
            <width>361</width>
            <height>281</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="addRecipe_2">
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>641</width>
            <height>471</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,1,1,1">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="4" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QRadioButton" name="Easy">
               <property name="styleSheet">
                <string notr="true">QRadioButton:checked{
    color: green;
}

QRadioButton:unchecked{
   color: grey;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="Medium">
               <property name="styleSheet">
                <string notr="true">QRadioButton:checked{
    color: orange;
}

QRadioButton:unchecked{
   color: grey;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="Hard">
               <property name="styleSheet">
                <string notr="true">QRadioButton:checked{
    color: red;
}

QRadioButton:unchecked{
   color: grey;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="5">
            <widget class="QSpinBox" name="spinBox"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Recipe Name:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="5" alignment="Qt::AlignRight">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Fav</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3" colspan="3">
            <widget class="QListWidget" name="listWidget"/>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="4">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Cooking Time:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="5">
            <widget class="QPushButton" name="createRecipe">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3" alignment="Qt::AlignLeft">
            <widget class="QSpinBox" name="spinBox_2">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>1</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>1000</width>
               <height>1000</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="3" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Ingredients: </string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Difficulty: </string>
             </property>
            </widget>
           </item>
           <item row="4" column="2" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Makes: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>157</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Instructions: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="addIngredient">
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>631</width>
            <height>481</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0" columnstretch="0,0,0,0,0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="spacing">
            <number>10</number>
           </property>
           <item row="5" column="3" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="createIngredient">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Add Ingredient</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Ingredient Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>1000</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QSpinBox" name="calAmount">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_3">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Calories: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Allergies: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="3" column="3" colspan="3">
            <widget class="QListWidget" name="viewAllergies">
             <property name="resizeMode">
              <enum>QListView::Adjust</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
